Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use the official Golang image
+FROM golang:1.22
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy go.mod and go.sum first to leverage Docker cache
+COPY go.mod go.sum ./
```

Comment:
This is not required considering the `COPY . .` directive is going to send `go.mod` and `go.sum` files inside the Docker Image